templates/events-list.php

```diff
@@ -3,16 +3,33 @@
 gp_tmpl_header();
 
 ?>
+<div class="event-list-top-bar">
+	<ul class="event-list-nav">
+		<li><a href="<?php echo esc_url( gp_url( '/events/my-events/' ) ); ?>">My Events</a></li>
+		<li><a class="button is-primary" href="<?php echo esc_url( gp_url( '/events/new/' ) ); ?>">Create Event</a></li>
+	</ul>
+</div>
 
 <h2 class="event_page_title">Upcoming Translation Events</h2>
+<div class="event-left-col">
 <?php
-
-
-$_paged = ( get_query_var( 'paged' ) ) ? get_query_var( 'paged' ) : 1;
-$args   = array(
+$current_datetime_utc = ( new DateTime( null, new DateTimeZone( 'UTC' ) ) )->format( 'Y-m-d H:i:s' );
```

Comment:
```suggestion
$current_datetime_utc = ( new DateTime('now', new DateTimeZone( 'UTC' ) ) )->format( 'Y-m-d H:i:s' );
```

Otherwise you get an error: Deprecated: DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated